onebusaway-android/src/main/java/org/onebusaway/android/ui/HomeActivity.java

```diff
@@ -425,6 +426,11 @@ private void goToNavDrawerItem(int item) {
                             getString(R.string.analytics_label_button_press_reminders));
                 }
                 break;
+            case NAVDRAWER_ITEM_PLAN_TRIP:
+                Intent planTrip = new Intent(HomeActivity.this, TripPlanActivity.class);
+                startActivity(planTrip);
+                // TODO - Analytics
```

Comment:
Still need to add analytics here.  Would be nice to add them elsewhere in the trip planning portion as well, although I'll need to look at that more carefully.